src/Illuminate/Database/Eloquent/Casts/AsFluent.php

```diff
@@ -20,7 +20,7 @@ public static function castUsing(array $arguments)
         {
             public function get($model, $key, $value, $attributes)
             {
-                return isset($value) ? new Fluent(Json::decode($value)) : null;
+                return new Fluent($value ? Json::decode($value) : []);
```

Comment:
Nullish coalescing operator might make more sense here:
```suggestion
                return new Fluent(Json::decode($value ?? '[]'));
```

Reply:
Accepted, thanks!